utopia-remix/.eslintrc.cjs

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
I'm assuming this was created by default when setting up the remix project? If so please can we bin this off and use the eslint config from the editor (https://github.com/concrete-utopia/utopia/blob/master/editor/.eslintrc-lite.js and https://github.com/concrete-utopia/utopia/blob/master/editor/.eslintrc.js)

Reply:
correct, it was default remix config. Should I just copypaste the other config or is there a way to link them (or do they even bubble down)?

Reply:
This is a bit gross, but we run the editor's config on all other projects like so https://github.com/concrete-utopia/utopia/blob/master/editor/package.json#L46